server/routerlicious/packages/services/src/redis.ts

```diff
@@ -44,18 +44,28 @@ export class RedisCache implements ICache {
 	}
 
 	public async get(key: string): Promise<string> {
-		return this.client.get(this.getKey(key));
+		try {
+			return this.client.get(this.getKey(key));
+		} catch (error) {
+			Lumberjack.error(`Error getting ${key} from cache.`, undefined, error);
+			return undefined;
+		}
 	}
 
 	public async set(key: string, value: string, expireAfterSeconds?: number): Promise<void> {
-		const result = await this.client.set(
-			this.getKey(key),
-			value,
-			"EX",
-			expireAfterSeconds ?? this.expireAfterSeconds,
-		);
-		if (result !== "OK") {
-			throw new Error(result);
+		try {
```

Comment:
Is this change in behavior intentional? This will cause us to continue processing a request after a failure to set something in cache. I think adding the log is a good idea, but I think we should `throw error` instead of `return undefined`

Reply:
Not intentional, fixed